webpack.common.config.js

```diff
@@ -104,7 +104,10 @@ module.exports = function (outputFilename) {
         // Babel
         {
           test: /\.js$/,
-          exclude: /node_modules/,
+          exclude: [
+            path.resolve(__dirname, "node_modules"),
```

Comment:
nit: singlequotes

Double checking - are we actually using the jquery autosuggest widget anywhere? If we do it this way, it means that we are including the file, but not running it through babel.

Reply:
Its used in views/forms/inputs/BasePicker in courage. Looks like we have a similar exclude list in courage webpack config.
```javascript
      loaders: [
        {
          test: /\.js$/,
          exclude: /(node_modules|vendor|okta\.js)/,
          loader: 'babel-loader',
          query: {
            presets: ['es2015'],
            cacheDirectory: '/tmp/.babel-cache'
          }
        }
      ]
```
@manueltanzi-okta made a similar fix for vkbeautify by setting it to empty `vendor/plugins/vkbeautify.0.99.00.beta': EMPTY`

The same fix seems to work for autosuggest i.e adding ti EMPTy list `'vendor/plugins/vkbeautify.0.99.00.beta': EMPTY`.

Reply:
Yeah, so that's the main question - is there any view that is using BasePicker with autosuggest? If there is nothing that uses it, we can go ahead with the empty fix (although once again, we should do it as high up as possible i.e. return empty for BasePicker rather than autosuggest). If there is a view that uses it, then we should go with the general fix, i.e. just ignore the vendor directory rather than calling it out explicitly.

Reply:
I did try moving basePicker to EMPTY list, although the build passes but i see an error when i try to load the widget on the browser. 

```javascript
Uncaught TypeError: BasePicker.extend is not a function
    at Object.<anonymous> (BaseSelectize.js:30)
    at Object.<anonymous> (BaseSelectize.js:1)
    at __webpack_require__ (bootstrap c9e73492f7024617fa23:19)
    at Object.<anonymous> (formsBundle.js:37)
    at __webpack_require__ (bootstrap c9e73492f7024617fa23:19)
    at Object.<anonymous> (viewsBundle.js:8)
    at __webpack_require__ (bootstrap c9e73492f7024617fa23:19)
    at Object.<anonymous> (Okta.js:16)
    at __webpack_require__ (bootstrap c9e73492f7024617fa23:19)
    at Object.<anonymous> (Util.js:14)
```
The error is coming from formsBundle view which is using BaseSelectize which extents BasePicker.